.github/workflows/azure_e2e_test.yml

```diff
@@ -0,0 +1,128 @@
+name: 'Azure E2E test'
```

Comment:
Note to reviewers: this workflow is still incomplete. I need to push the workflow to main branch to be able to test it.